feature/search/src/main/kotlin/uk/govuk/app/search/api/SearchRepo.kt

```diff
@@ -0,0 +1,39 @@
+package uk.govuk.app.search.api
+
+import uk.govuk.app.search.api_result.SearchResponse
+import uk.govuk.app.search.domain.ResultStatus
+import uk.govuk.app.search.domain.SearchConfig
+import uk.govuk.app.search.domain.SearchResult
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class SearchRepo @Inject constructor(
+    private val searchApi: SearchApi
+) {
+
+    suspend fun initSearch(
```

Comment:
Can we rename this to performSearch, getSearchResults, fetchSearchResults or something similar?